interface/bower.json

```diff
@@ -21,10 +21,13 @@
     "ng-sortable": "~1.3.6",
     "ng-vega": "^1.3.2",
     "d3-tip": "~0.7.1",
-    "angularjs-slider": "^5.8.1"
+    "angularjs-slider": "^5.8.1",
+    "vega-lite": "^1.3.1",
```

Comment:
This more of a clarification point for me: So we need the vega-lite library even though we have already loaded vega itself?